### DISCUSSION TITLE
Flet run outputs error, Fedora 39

### DISCUSSION THREAD

**Discussion by nexusvoyager** [flet-dev/flet#2696] [Q&A]
Howdy! I've recently installed flet to give it a shot in developing my software, however everytime i run flet I got the following error:

```
solidsnake@fedora:~/phantom-flet$ flet run
/home/solidsnake/.flet/bin/flet-0.20.2/flet/flet: error while loading shared libraries: libmpv.so.1: cannot open shared object file: No such file or directory
```
It wants me to install `libmpv`, so i looked through the fedora repositories and looked for the `libmpv` package, it was called `mpv-libs` rather than `libmpv`, so installed `mpv-libs`, `mpv-devel`, and `mpv` itself, tried to run the command, nothing worked, any advice on what I can do? I would appreciate some help with this  :D

**Comment by emojiiii** [comment]
https://github.com/flet-dev/flet/discussions/2639

**Comment by tovganesh** [comment]
@planetjunkie 

I have the same issue with Fedora 39. Installed the mpv-libs and mpv-devel but this error persists. 

A temporary workaround is to add the following symlink:

sudo ln -s /usr/lib64/libmpv.so.2 /usr/lib64/libmpv.so.1

After adding this symlink, I am able to run the flet apps.

**Reply by nexusvoyager** [reply]
I've run this, and now it has gave me the following error
`/home/solidsnake/.flet/bin/flet-0.20.2/flet/flet: error while loading shared libraries: libmedia_kit_libs_linux_plugin.so: cannot open shared object file: No such file or directory`

**Reply by nexusvoyager** [reply]
Nevermind, I fixed it, just needed to reinstall flet :p

**Comment by luizmarinhojr** [comment]
I'm using the Fedora 39

I tried to reinstall the packages and also the flet, but didn't works.

I found a another solution and it worked for me. The solution is bellow.

### Installing the pakages mpv-libs and mpv-devel on Fedora 39

Run the command to install the packages:
`sudo dnf install mpv-libs`
`sudo dnf install mpv-devel` 

### Searching the packages on directory

`cd /usr/lib64`

Searching the mpv
`find *mpv*`

The answer must be:
`libmpv.so
libmpv.so.2
libmpv.so.2.1.0
`

### Creating a symbolic link
`sudo ln -s /usr/lib64/libmpv.so /usr/lib64/libmpv.so.1`

Done.

**Reply by ArielleFox** [reply]
I was already thinking of setting up another distro just to avoid these fedora headaches even tho I like this distro thank you <3
